lib/spandex_phoenix/telemetry.ex

```diff
@@ -0,0 +1,34 @@
+defmodule SpandexPhoenix.Telemetry do
+  @moduledoc """
+  Defines
+  """
+
+  @doc "Attaches `:telemetry` event handlers for Phoenix Telemetry events."
```

Comment:
todo - docs re: you can pass the tracer at runtime here instead of relying on application config